### ISSUE TITLE
Enhancement: More difficulty adjustments per difficulty, and modified normal charts.

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#2618]
## What is your suggestion, and why should it be implemented?
I always thought having to play on hard mode to get the full chart was a little bit weird. I think normal mode should still offer the full chart, just maybe with simpler patterns and less doubles. Alternatively normal could be like hard and then hard mode could be buffed further of course. Instead of large differences in charting, I think it would be more practical for the majority of the difficulty shift to instead come from mechanics. For example, simple stuff like losing more HP upon missing, or actual mechanics like having more can throws in 2Hot, hitting a can shot having a smaller timing window, and missing a can dealing more damage to you, etc. This would make difficulty more than just one charting change and make it so that players don't feel like they're not getting the full experience by playing on normal, as normal is now a full chart, just with easier mechanics and easier to hit patterns But no full notes are missed.

**Comment by oddMLan** [comment]
You articulated my same thoughts very accurately. Hear this guy out devs!

**Comment by Average-FNF-Modder** [comment]
Oh fuck yeah

**Comment by Hundrec** [comment]
It's kind of an edge case
There were a *lot* of recharts and adjustments, but probably not exactly as you suggested
You can create a new issue based on the new charts (or wait until 0.6.3's chart changes)

**Comment by Average-FNF-Modder** [comment]
> It's kind of an edge case There were a _lot_ of recharts and adjustments, but probably not exactly as you suggested You can create a new issue based on the new charts (or wait until 0.6.3's chart changes)

I mean, I'm more sad at the lack of mechanical difficulty change implementations

**Comment by oddMLan** [comment]
What OP meant is that some song charts in Normal and Erect difficulties flat out omit notes for the sake of having less inputs despite not matching the vocals entirely. They suggest to instead focus on on making simpler patterns that STILL match ALL of the vocals, just slightly less crazy patterns. Difficult patterns such as "jacks", "stairs", etc, be replaced with alternatives that don't require shifting your fingers as quickly? (Hard to describe without a visual example to be fair).
This is NOT the same as using less inputs, just simpler patterns. I think this would alleviate the demands for recharting some songs with wicked diffuclt patterns like Spookees Nightmare ver. I believe Nightmare needs to honor its name, and keep the charts a lil' crazy. People like myself just don't bother with the Erect difficulty when Nightmare is the actual full chart that 100% matches the vocals. I find Nightmare a challenge, and I'd be happy to play in Erect if it didn't 'dumb down' the chart that much by not matching the vocals all the way.

Besides increasing the difficulty of patterns, there are other ways to increase the overall difficulty. Such as
- Higher HP drain for missing notes + lower HP recovery for Good/Bad/Shit rankings in Hard and Nightmare modes. Normal and Erect modes, keep as-is.
- Tighter timings for gameplay mechanics such as shooting the cans in 2hot (hitting it "bad" should drain health in Hard/Night modes, "shit" even more, and missing it altogether should drain like half of your health). This philosophy would also extend to any future gameplay mechanics.

**Comment by Hundrec** [comment]
Yes, I actually think easier difficulties should be exactly that: simpler patterns without undercharting. Undercharting is awful!
I marked this as resolved because the charter did so, but I'll reopen this now.

**Comment by Average-FNF-Modder** [comment]
> What OP meant is that some song charts in Normal and Erect difficulties flat out omit notes for the sake of having less inputs despite not matching the vocals entirely. They suggest to instead focus on making simpler patterns that STILL match ALL of the vocals, just slightly less crazy patterns. Difficult patterns such as "jacks", "stairs", etc, be replaced with alternatives that don't require shifting your fingers as quickly? (Hard to describe without a visual example to be fair). This is NOT the same as using less inputs, just simpler patterns. I think this would alleviate the demands for recharting some songs with wicked difficult patterns like Spookees Nightmare ver. I believe Nightmare needs to honor its name, and keep the charts a lil' crazy. People like myself just don't bother with the Erect difficulty when Nightmare is the actual full chart that 100% matches the vocals. I find Nightmare a challenge, and I'd be happy to play in Erect if it didn't 'dumb down' the charts that much by not matching the vocals all the way.
> 
> Besides increasing the difficulty of patterns, there are other ways to increase the overall difficulty. Such as
> 
> * Higher HP drain for missing notes + lower HP recovery for Good/Bad/Shit rankings in Hard and Nightmare modes. Normal and Erect modes, keep as-is.
> * Tighter timings for gameplay mechanics such as shooting the cans in 2hot (hitting it "bad" should drain health in Hard/Night modes, "shit" even more, and missing it altogether should drain like half of your health). This philosophy would also extend to any future gameplay mechanics.

Thank you for specifying my ideas so nicely

**Comment by oddMLan** [comment]
> Besides increasing the difficulty of patterns, there are other ways to increase the overall difficulty. Such as

Just thought of another one: Hold notes. Holding notes for less than 50% of their duration should combo break in Hard and Night modes. Could be like 
>%75 = SICK. 
74-51% = GOOD.
Combo Breaking:
<50% = BAD
<25% = SHIT

This would of course mean that for every hold note there would be two input events: landing the arrow note, and releasing it on time. As a bonus, that would increase the number of inputs in the score screen too! And without undercharting the easier modes!
In Normal mode, I would probably keep it as it currently is. No penalty for messing up hold notes.

For this ticket though, I would probably limit the scope to just addressing the undercharting of Normal and Erect difficulties.

**Comment by Hundrec** [comment]
Looks like undercharting's here to stay

**Comment by Hundrec** [comment]
It appears so! This issue will always stay on the tracker and can always be revisited, but the charters marked this as rejected.
